c++/src/kj/exception.c++

```diff
@@ -1047,6 +1054,7 @@ public:
     text = str(kj::repeat('_', contextDepth), file, ":", line, ": ", severity, ": ",
                mv(text), '\n');
 
+#ifndef KJ_NO_POSIX
```

Comment:
I guess logs just won't go anywhere unless the application defines an ExceptionCallback? This seems kind of bad as those logs are usually important for debugging problems... I worry about people not even realizing that they're missing the logs.

Reply:
> It would be nice to have some sort of a CI test to verify that this mode actually does compile, otherwise we'll surely break it. 

Agreed, will do.

> It would be even nicer to have unit tests... but I suppose it's pretty hard to define unit tests when you don't even have a standard output for writing a test log. What is the standard practice here?

We can compile and test the unit tests on the host system.  I'll work on something.